### DISCUSSION TITLE
удаление

### DISCUSSION THREAD

**Discussion by pogchampis** [bol-van/zapret#887] [Announcements]
у меня такая проблема что я через батник discord поставил запрет и хочу теперь убрать а через ремув выдает ошибку 1060 через cmd неправильный ввод либо пусто пробовал через права админа 
вводил команды и тд ничего не помагает

**Comment by truechupa** [comment]
Зачем удалять вопросы? Ведь это тоже своего рода база знаний.

**Comment by electrifying** [comment]
Какие батники, в этом репозитории их вообще нет. Открывайте иссусы там, где скачивали
